solr/core/src/test/org/apache/solr/response/TestCustomDocTransformer.java

```diff
@@ -26,7 +26,6 @@
 import org.apache.solr.request.SolrQueryRequest;
 import org.apache.solr.response.transform.DocTransformer;
 import org.apache.solr.response.transform.TransformerFactory;
-import org.bouncycastle.util.Strings;
```

Comment:
Test used to use old bouncycastle dependency which isn't needed anymore from Hadoop. Switched to use builtin Java split.